tests/test_costfunction.py

```diff
@@ -1,26 +1,25 @@
 import sys
-import unittest
 
 import numpy as np
 
 sys.path.append('./shared')
-import activationfunction as af
-
-class Test_Cost_Function(unittest.TestCase):
-
-    def setUp(self):
-        # init
-        pass
-
-    def tearDown(self):
-        # dispose
-        pass
-
-    def test_rss(self):
-        pass
-
-    def test_cross_entropy(self):
-        pass
-
-if __name__ == "__main__":
-    unittest.main()
\ No newline at end of file
+import costfunction
+
+def setUp():
+    # init
+    pass
+
+def tearDown():
+    # dispose
+    pass
+
+def test_rss():
+    pass
+
+def test_cross_entropy():
+    label = np.array([0, 0, 1, 1])
+    ans = np.array([0.2, 0.3, 0.4, 0.5])
+    error = costfunction.cross_entropy(label, ans)
+    assert error - 2.189 < 1e-3, "check cross_entropy or test"
```

Comment:
1. Can you divide into to two tests, `test_cross_entropy()` and `test_diff_cross_entropy()`?


2. Please Use `np.isclose()`.
```
# This passed test
assert error - 2000000.189 < 1e-3, "check cross_entropy or test"
```

Reply:
1 yes
2 yes